packages/ui/src/components/va-dropdown/VaDropdown.vue

```diff
@@ -174,14 +174,15 @@ export default defineComponent({
     })
 
     const anchorComputed = computed(() => {
-      return props.cursor ? cursorAnchor.value : anchor.value
+      if (!props.cursor) { return anchor.value }
+
+      return useCursorAnchor(anchor, valueComputed).value
     })
```

Comment:
Composables mustn't be used outside setup() execution flow. useCursorAnchor will create computed on each recalculation of anchorComputed. Computed stores effect globally, so there might be potential memory leaks.

```ts
    const cursor = useCursorAnchor(anchor, valueComputed)

    const anchorComputed = computed(() => {
      if (!props.cursor) { return anchor.value }
      return cursor.value
    })
```